src/Spago/Dhall.hs

```diff
@@ -41,6 +41,17 @@ prettyWithHeader header expr = do
   let doc = Pretty.pretty header <> Pretty.pretty expr
   PrettyText.renderStrict $ Pretty.layoutSmart Pretty.defaultLayoutOptions doc
 
+stripComments :: String -> Text -> Text
+stripComments file dhallSrc =
+  case Parser.exprFromText file dhallSrc of
+    -- TODO parsing of template could potentially fail (e.g. with update of dhall version)
+    -- what to do in that case? Print a warning and return not-stripped template?
+    Left _ -> dhallSrc
```

Comment:
Is this approach acceptable, or should I print some warning in Spago monad?

Reply:
This approach is what I would implement myself, great by me 🙂